scripts/run-ci-local.js

```diff
@@ -1,60 +1,36 @@
 const { execSync } = require('child_process');
 const { renameSync } = require('fs');
 const { join, basename } = require('path');
+const { mkdirpSync } = require('fs-extra');
 
 const cwd = process.cwd();
 
 const nodeModulesDir = join(cwd, 'node_modules');
-const nodeModulesDevDir = join(cwd, '.node_modules__dev');
 const nodeModulesDockerDir = join(cwd, '.node_modules__docker');
+//const dockerImage = 'uxaspects/buildenv:latest';
+const dockerImage = 'cafinternal/prereleases:buildenv-1.3.0-EL-3727-SNAPSHOT';
 
-try {
-    // Move existing node_modules to a known location
-    renameSync(nodeModulesDir, nodeModulesDevDir);
-    console.log(`Renamed node_modules to ${basename(nodeModulesDevDir)}. You can use this instead of doing \`npm ci\` again after exiting the shell.`);
-} catch (error) {
-    if (error.code !== 'ENOENT') {
-        console.warn(`Couldn't rename node_modules. (The IDE probably has a handle on it.)`);
-    }
-}
+// Ensure the docker node_modules__docker directory exists
+mkdirpSync(nodeModulesDocker);
```

Comment:
This should be `nodeModulesDockerDir` (or rename the above variable).

Reply:
Argh, still getting used to VS Code and it not auto saving :)

Reply:
There is an auto-save setting, although I can't vouch for it:
![image](https://user-images.githubusercontent.com/26114101/69814617-7e787900-11ec-11ea-8e9d-5c442df9c98b.png)